Libplanet.Headless/Hosting/LibplanetNodeService.cs

```diff
@@ -313,6 +313,12 @@ public override async Task StopAsync(CancellationToken cancellationToken)
                         maxLogFileSize: 16 * 1024 * 1024,
                         keepLogFileNum: 1
                     );
+                    if (RocksDBStore.RocksDBStore.MigrateChainDBFromColumnFamilies(
```

Comment:
Is this call idempotent (in other words, okay if it's invoked more than once)?

Reply:
Yes, it checks whether it is migrated first and does nothing if it is.